### PR TITLE
Liam/gh issue 9

### PR DESCRIPTION
**Pull request by elmdecoste**

### CODE REVIEW

File: static/js/components/CardGrid.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+export default class CardGrid extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    componentDidMount() {
+        fetch('http://localhost:5000/api/items/all')
```
Comment (simplyluke) [inline static/js/components/CardGrid.js]: this should be abstracted from localhost to the current base url

File: static/js/components/CardGrid.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import CardComponent from "./CardComponent"
+
+require('../../css/CardComponent.scss');
```
Comment (simplyluke) [inline static/js/components/CardGrid.js]: Fine with it for now but lets make an issue to revisit this part of the scss approach, not a fan

File: static/css/CardComponent.scss [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
+}
+
+//10px Gutter
+.gutter-8.row {
```
Comment (simplyluke) [inline static/css/CardComponent.scss]: what are these gutter classes doing and why is bootstrap's default behavior not acceptable?

File: static/package.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "scripts": {
+    "build": "webpack -p --progress --config webpack.config.js",
+    "dev-build": "webpack --progress -d --config webpack.config.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --progress -d --config webpack.config.js --watch"
+  },
+  "babel": {
+    "presets": [
+      "es2015",
```
Comment (simplyluke) [inline static/package.json]: Any reason to use es2015 over es6?

File: templates/index.html [hunk 5]
```diff
@@ -1,9 +1,55 @@
...
+{% block head %}
+    {{ super() }}
+    <link href='{{ url_for('static', filename='css/index.scss') }}' rel='stylesheet' type='text/css'>
+{% endblock %}
 
 {% block page_content %}
-<div class="page-header">
-    <h1>Hello World!</h1>
-</div>
+    <div class="row">
```
Comment (simplyluke) [inline templates/index.html]: is this wrapped in a "container" or "container-fluid" somewhere? Will cause headaches later if not.

Comment (simplyluke) [reply templates/index.html]: never mind - carousel

File: static/css/CardComponent.scss [hunk 6]
```diff
@@ -0,0 +1,87 @@
+@mixin aspect-ratio($width, $height) {
```
Comment (simplyluke) [inline static/css/CardComponent.scss]: making a ticket and assigning myself, feel free to assign yourself as well. Want to use the scss file structure I've found beneficial to organize stylesheets on this proj and allow them to be compiled to one minified file. Fine with this for the time being though

File: static/css/CardComponent.scss [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+  .card-title-contents {
+    display: flex;
+    align-items: center;
+    justify-contents: flex-end;
+  }
+}
+
+body {
+  background-color: #E1E1E1;
```
Comment (simplyluke) [inline static/css/CardComponent.scss]: Fine for now, we'll want to abstract things like colors to scss variables though

File: static/css/CardComponent.scss (hunk 3 above)
Comment (elmdecoste) [reply static/css/CardComponent.scss]: The bootstrap gutters were larger than desired so we updated them to be smaller

File: static/css/CardComponent.scss (hunk 7 above)
Comment (elmdecoste) [reply static/css/CardComponent.scss]: Let's make an issue and assign that to both you and I, I'd like to see how to do that

File: static/js/components/CardGrid.js (hunk 1 above)
Comment (elmdecoste) [reply static/js/components/CardGrid.js]: Yes, that was an oversight

File: static/package.json (hunk 4 above)
Comment (elmdecoste) [reply static/package.json]: Not really, we can use ES6 if we want to

Comment (simplyluke) [reply static/package.json]: Kk fine for now we can use es6 if we want later, adds some cool typing

File: static/css/CardComponent.scss (hunk 7 above)
Comment (simplyluke) [reply static/css/CardComponent.scss]: kk will do

File: static/js/components/CardGrid.js (hunk 1 above)
Comment (simplyluke) [reply static/js/components/CardGrid.js]: will make an issue about this

**Comment by simplyluke** [comment]
Making issues for what we discussed, looks good to me

**Pull request merged**